python/lsst/ap/association/afwUtils.py

```diff
@@ -230,29 +234,31 @@ def make_dia_object_schema(filter_names=None):
     #                     'periodogram for y filter. [20 FLOAT].')
     schema.addField('nearbyObj1', type='L',
                     doc='Id of the closest nearby object.')
-    schema.addField('nearbyObj1Dist', type='D',
+    schema.addField('nearbyObj1Dist', type='F',
                     doc='Distance to nearbyObj1.')
-    schema.addField('nearbyObj1LnP', type='D',
+    schema.addField('nearbyObj1LnP', type='F',
                     doc='Natural log of the probability that the observed diaObject is the same as the '
                         'nearbyObj1.')
     schema.addField('nearbyObj2', type='L',
                     doc='Id of the second-closest nearby object.')
-    schema.addField('nearbyObj2Dist', type='D',
+    schema.addField('nearbyObj2Dist', type='F',
                     doc='Distance to nearbyObj2.')
-    schema.addField('nearbyObj2LnP', type='D',
+    schema.addField('nearbyObj2LnP', type='F',
                     doc='Natural log of the probability that the observed diaObject is the same as the '
                         'nearbyObj2.')
     schema.addField('nearbyObj3', type='L',
                     doc='Id of the third-closest nearby object.')
-    schema.addField('nearbyObj3Dist', type='D',
+    schema.addField('nearbyObj3Dist', type='F',
                     doc='Distance to nearbyObj3.')
-    schema.addField('nearbyObj3LnP', type='D',
+    schema.addField('nearbyObj3LnP', type='F',
                     doc='Natural log of the probability that the observed diaObject is the same as the '
                         'nearbyObj3.')
     schema.addField('flags', type='L',
                     doc='Flags, bitwise OR tbd.')
     schema.addField('pixelId', type='L',
                     doc='HTM index.')
+    schema.addField('lastNonForcedSource', type='L',
+                    doc='Last time when non-forced DIASource was seen for this object')
```

Comment:
I'm not sure that you want this column on your AFW schema - this is an implementation detail internal to dax_ppdb and you are not supposed to depend on it, I think this schema should only depend on things that are specified in PPDB. I guess you want AFW schema returned from database to be exactly the same as what you saved? If you cannot tolerate few extra columns in returned AFW schema then we should filter out those internal columns from returned result.

And I think I'm OK with this change for now if that helps you fixing issues but you should be aware that dax_ppdb internals are not stable and be prepared to update this schema if those internals change.

Reply:
See the below comment. I don't really need these columns to exist, but as is can be returned by the ppdb so to make the schemas match I include it here.